### PR TITLE
adds /dev/shm workaround to Docker common problems section in CI 

### LINKED ISSUE #2621: Add /dev/shm Docker workaround to Continuous Integration Docker problems section
## I'm submitting a...

```
[ ] Bug report
[X] Content update
[ ] Process update (build, deployment, ... )
```

## Type of bug / changes

Add documentation about size of the dev/shm within a docker container and the workaround from the comments here: https://github.com/cypress-io/cypress/issues/5336

### PR DESCRIPTION
**Pull request by CypressCecelia**
Closes #2621

### CODE REVIEW

File: source/guides/guides/continuous-integration.md [hunk 1]
```diff
@@ -700,6 +700,38 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
...
+  on('before:browser:launch', (browser = {}, args) => {
+    if (browser.family === 'chrome') {
+      console.log('Adding --disable-dev-shm-usage...')
+      args.push('--disable-dev-shm-usage')
+    }
+
+    return args
+  })
+}
+```
```
Comment (jennifer-shehane) [inline source/guides/guides/continuous-integration.md]: I would not document anything in the docs that's pre-current version, it just invites a precedent that we do not intend to maintain (unless we explicitly create versioned docs one day). This is reserved for the migration guide + history tabs + changelog.
```suggestion
```

File: source/guides/guides/continuous-integration.md [hunk 2]
```diff
@@ -700,6 +700,38 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
 
 If you are running long runs on Docker, you need to set the `ipc` to `host` mode. {% issue 350 'This issue' %} describes exactly what to do.
 
+In a Docker container, the default size of the `/dev/shm` shared memory space is 64MB. This is not typically enough to run Chrome and can cause the browser to crash. You can fix this by passing the `--disable-dev-shm-usage` flag to Chrome with the following workarounds:
+
+### Cypress versions 4.0.0 and up
```
Comment (jennifer-shehane) [inline source/guides/guides/continuous-integration.md]:
```suggestion
```

**Comment by CypressCecelia** [comment]
Thanks! Committed suggested changes.

File: source/guides/guides/continuous-integration.md [hunk 3]
```diff
@@ -700,6 +700,23 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
 
 If you are running long runs on Docker, you need to set the `ipc` to `host` mode. {% issue 350 'This issue' %} describes exactly what to do.
 
+In a Docker container, the default size of the `/dev/shm` shared memory space is 64MB. This is not typically enough to run Chrome and can cause the browser to crash. You can fix this by passing the `--disable-dev-shm-usage` flag to Chrome with the following workarounds:
+
+
```
Comment (jennifer-shehane) [inline source/guides/guides/continuous-integration.md]:
```suggestion
In a Docker container, the default size of the `/dev/shm` shared memory space is 64MB. This is not typically enough to run Chrome and can cause the browser to crash. You can fix this by passing the `--disable-dev-shm-usage` flag to Chrome with the following workaround:

```

File: source/guides/guides/continuous-integration.md [hunk 4]
```diff
@@ -700,6 +700,23 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
...
+```javascript
+module.exports = (on, config) => {
+  on('before:browser:launch', (browser = {}, launchOptions) => {
+    if (browser.family === 'chrome') {
```
Comment (jennifer-shehane) [inline source/guides/guides/continuous-integration.md]: So I didn't run this code before, but running this is actually inaccurate. It should be:
```suggestion
    if (browser.family === 'chromium' && browser.name !== 'electron') {
```

File: source/guides/guides/continuous-integration.md [hunk 5]
```diff
@@ -700,6 +700,23 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
...
+      console.log('Adding --disable-dev-shm-usage...')
```
Comment (jennifer-shehane) [inline source/guides/guides/continuous-integration.md]: I would remove this console.log - just going to be confusing to people who copy/pasta - where this extra log is coming from.
```suggestion
```

File: source/guides/guides/continuous-integration.md [hunk 6]
```diff
@@ -700,6 +700,23 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
...
+      launchOptions.args.push('--disable-dev-shm-usage')
+    }
+
+    return launchOptions
+  })
+}
+```
+
+
```
Comment (jennifer-shehane) [inline source/guides/guides/continuous-integration.md]: extra line
```suggestion

```

**Pull request merged**